extensions/arc/deployment/src/main/resources/dev-ui/qwc-arc-removed-components.js

```diff
@@ -45,6 +47,22 @@ export class QwcArcRemovedComponents extends LitElement {
         this._removedInterceptors = removedInterceptors;
     }
 
+    connectedCallback() {
+        super.connectedCallback();
+
+        this.jsonRpc.getInactiveBeans().then(inactiveBeans => {
```

Comment:
Hm, the `jsonRpc.getInactiveBeans()`, i.e. `Arc.container().beanManager().getBeans()`, will never return a removed bean... In other words, `bean.inactive` will be always `false` for removed beans.

Reply:
Humm, good point. I didn't really think about it, but in hindsight, trying to show an "inactive" status for removed beans is nonsense. That is never gonna happen. Will remove.

Reply:
Removed, thanks for noticing!